### ISSUE TITLE
bug: eza -FT no longer works

### ISSUE THREAD

**Issue by Jab2870** [eza-community/eza#832]
```
$ eza --version
eza - A modern, maintained replacement for ls
v0.18.0 [+git]
https://github.com/eza-community/eza
```

As the title says:

```
$ eza -FT
eza: Flag -F cannot take a value
```

This previously worked. `eza -TF` does work, but mussel memory is the other way around

**Comment by varunpalekar** [comment]
I also face same of `la` alias

**Comment by MartinFillon** [comment]
Well, this is mostly due to the current parser, @cafkafk we kinda need clap to be merged asap. As it would resolves issuess like that.

**Comment by jmnc2** [comment]
I also get the same problem with 'exa -aalFh'

**Comment by salkin-mada** [comment]
Yeah it seems that somewhere between version 0.17.3 (homebrew) and 0.18.17 (arch) `ls` compatibility is broken (in regards to the positioning of flag `-F`.
@MartinFillon is there work being done on merging clap? `branch-x`?

**Comment by MartinFillon** [comment]
YEah so it got broked when --classify (aka -F) started taking optional args.
As per clap I am waiting for @cafkafk review

I am trying my best in order to merge everything in the best way possible.

**Comment by J-Siu** [comment]
> Yeah it seems that somewhere between version 0.17.3 (homebrew) and 0.18.17 (arch) `ls` compatibility is broken (in regards to the positioning of flag `-F`). @MartinFillon is there work being done on merging clap? `branch-x`?

Yes. It only work if `F` is the last option. Example:
```
eza -FGH <-- Return "eza: Flag -F cannot take a value"
eza -GHF <-- Works correctly
```
